mlir/include/mlir/IR/Builders.h

```diff
@@ -334,6 +334,18 @@ class OpBuilder : public Builder {
   /// This class represents a saved insertion point.
   class InsertPoint {
   public:
+    /// Finds the closest insertion point where all given values are defined
```

Comment:
It's not clear to me that this is a well defined problem: what does "closest means" here?

Reply:
Possible candidates are the points (`Block::iterator`) right after the definition of each value. I will take a look at all of these and find the one that post-dominates all the others. I think there should be only one such point. Will update this PR accordingly soon...